codespell_lib/data/dictionary.txt

```diff
@@ -3716,6 +3722,10 @@ codepoitn->codepoint
 codesc->codecs
 codespel->codespell
 coditions->conditions
+coduct->conduct
+coducted->conducted
```

Comment:
Add `coductor->conductor` and probably `coducter->conductor`

Reply:
900c7b3